### PR TITLE
39124: Update content and styling

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ﻿@model NHSD.BuyingCatalogue.Identity.Api.ViewModels.Consent.ConsentViewModel
 
 @{
-    ViewData["PageTitle"] = "User agreement";
-    ViewData["PageDescription"] = "PLEASE READ THESE TERMS CAREFULLY BEFORE USING THE CATALOGUE";
+    ViewData["PageTitle"] = "Buying Catalogue End User Agreement";
+    ViewData["PageDescription"] = "Please read this agreement carefully before using the NHS Digital Buying Catalogue";
 }
 
 <style>
```
Comment (kat1906) [inline src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml:8]: I am not sure about these styles being in her

Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml:8]: For the styles do you think its better pulling them out into a seperate css class?

Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml:8]: @kat1906  ^

Comment (kat1906) [reply src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml:8]: I personally don't mind as it is unlikely they will get used elsewhere and the li style should just be applied on this page

Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml:8]: Fair enough. Keep it in then 👍

**Pull request merged**